generator/src/test/java/com/scottlogic/deg/generator/fieldspecs/relations/EqualToOffsetDateRelationTest.java

```diff
@@ -0,0 +1,56 @@
+package com.scottlogic.deg.generator.fieldspecs.relations;
+
+import com.scottlogic.deg.common.profile.Field;
+import com.scottlogic.deg.generator.fieldspecs.FieldSpec;
+import com.scottlogic.deg.generator.restrictions.DateTimeRestrictions;
+import org.junit.jupiter.api.Test;
+
+import java.time.OffsetDateTime;
+import java.time.Period;
+import java.time.ZoneOffset;
+import java.time.temporal.TemporalAdjuster;
+import java.time.temporal.TemporalAmount;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class EqualToOffsetDateRelationTest {
+
+    @Test
+    public void testReduce_isEqualTo_OtherField() {
```

Comment:
Not a big deal, but normally our tests have been of the form `<method>_<scenario>_<expectation>`. It would be good to see that here.